Casks/s/slidepilot.rb

```diff
@@ -9,7 +9,9 @@
 
   livecheck do
```

Comment:
```suggestion
  # The Sparkle feed contains `pubDate` values that are in German (e.g.
  # Fr., 05 März 2021 10:29:18 +0100), so the `Sparkle` strategy doesn't
  # accurately sort the items by date. We have to work with all the feed items
  # in the `strategy` block, as a way of avoiding the sorting issues.
  livecheck do
```

Using `items` (instead of `item`) in a `Sparkle` `strategy` block is a special case, so I make sure to document them. This helps us to understand why it's necessary, in case the situation ever changes in the future.

Reply:
Thanks!